v2_1/ws/rest/docs/4_4_data_queries.md

```diff
@@ -1,91 +1,104 @@
+- All parameters support multiple values, using comma (`,`) as separator.
+- All parameters offer the option to retrieve all existing values, using the `*` operator.
+- Two additional operators are supported for the version parameter: the `+`, to indicate the latest stable version of an artefact, and `~`, to indicate the latest version of an artefact regardless of its status (draft vs. stable). 
+- For all path parameters, the default value is `*`.
+- Default values do not need to be supplied if they are the last element in the path.
 
-#### Parameters used to further filter the desired results
+#### Query parameters
 
-The following parameters are used to further describe (or filter) the desired results, once the resource has been identified. As mentioned in 3.2, these parameters go in the query string part of the URL.
+The following query parameters are supported:
 
 Parameter | Type | Description
 --- | --- | ---
-startPeriod | common:StandardTimePeriodType, as defined in the SDMXCommon.xsd schema. Can be expressed using *dateTime* (all data that falls between the calendar dates will be matched), *Gregorian Period* (all data that falls between the calendar dates will be matched) or *Reporting Period* (all data reported as periods that fall between the specified periods will be returned. When comparing reporting weeks and days to higher order periods (e.g. quarters) one must account for the actual time frames covered by the periods to determine whether the data should be included. Data reported as Gregorian periods or distinct ranges will be returned if it falls between the specified reporting periods, based on a reporting year start day of January 1). In case the `:` or `+` characters are used, the parameter must be [percent-encoded](http://en.wikipedia.org/wiki/URL_encoding#Percent-encoding_reserved_characters) by the client. Note that this value is assumed to be inclusive to the range of data being sought. For additional information about the formats, see section 4.2.14 of Section 06 (SDMX Technical Notes). | The start period for which results should be supplied (inclusive).
-endPeriod | Same as above | The end period for which results should be supplied (inclusive).
+c | Map | Filter data by component value. For example, if a structure defines a frequency dimension (FREQ) and the code A (Annual) is an allowed value for that dimension, the following can be used to retrieve annual data: `c[FREQ]=A`. The same applies to attributes (e.g. `c[CONF_STATUS]=F`) and measures. Multiple values are supported, using a comma (`,`) as separator: `C[FREQ]=A,M`. In case of attributes that support multiple values, the plus (`+`) can be used to list all values that an attribute must have. For example, to indicate that ATTR1 must either be A or B AND M, use the following: `C[ATTR1]=A,B+M`. Operators may be used too (see table with operators below). This parameter can be used in addition, or instead of, the `key` path parameter. 
```

Comment:
Same comment as for availability: C --> c, "A or B AND M" --> "A or (B AND M)", multiple c's.

Reply:
Thanks, I have made the proposed changes.